docs/requirements.txt

```diff
@@ -3,10 +3,10 @@
 
 # Learn from https://github.com/rusty1s/pytorch_geometric/blob/master/docs/requirements.txt
 # Remove if switching to DGL
```

Comment:
Is this comment (line 5) still valid with the following lines commented off (but we are not switching to DGL)? Discuss with Peizhen (who wrote this)  if needed.

Reply:
@xianyuanliu @peizhenbai Any suggestions for this problem?

Reply:
The Cython part is not annotated now. It mignt be caused by some operations in developing.